lib/redis_memo/options.rb

```diff
@@ -9,7 +9,11 @@ def initialize(
     redis_error_handler: nil,
     tracer: nil,
     global_cache_key_version: nil,
-    expires_in: nil
+    expires_in: nil,
+    max_connection_attempts: nil,
+    disable_all: false,
+    disable_cached_select: false,
+    disabled_models: Set.new
```

Comment:
I was thinking that when we change traject to use the new version, we should read the env vars from there and pass them into the options rather than read them from the redis-memo repo.

Reply:
Ah right. For people who don't have those configured in their initializers, would it be valuable to still have the env vars as a fallback?

Reply:
Would it be better to just have one place and one way to configure RedisMemo options though? Different users might want to name their env variables differently or pull the values from a different source.

For traject, I think as long as we map the environment variables to the right options and change it at the same time as the version is being bumped, it shouldn't break anythning?